src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -22,7 +22,8 @@
  */
 public class ParserUtil {
 
-    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_INDEX =
+            "Index is not a non-zero unsigned integer from 1 to Integer.MAX_VALUE";
```

Comment:
Would it be better to use 2147483647 instead for those unfamiliar with Java syntax?
Also cos iirc some integer max value is different, like 32767 / depends on system

Reply:
hehexd

Reply:
i think ill change it to 2^31 - 1? would that be better than just a string of numbers